airflow/contrib/hooks/sagemaker_hook.py

```diff
@@ -232,15 +234,20 @@ def get_conn(self):
 
     def get_log_conn(self):
         """
-        Establish an AWS connection for retrieving logs during training
+        This method is deprecated. Please use `airflow.contrib.hooks.AwsLogsHook.get_conn` instead.
```

Comment:
```suggestion
        This method is deprecated. Please use :py:meth:`get_conn` instead.
```

Reply:
I'm actually referencing a method in another module so I changed it to
```
:py:meth:`airflow.contrib.hooks.AwsLogsHook.get_conn`
```

Is that the correct way?

Reply:
If it's in the same class then the suggestion I put would work.

You should be able to test this yourself by running `make -C docs html && open docs/_build/html/index.html`

Reply:
It's not in the same module though, it it 🤦‍♂ This is probably right as you have it.

Reply:
Haha it's in a different one yeah. I did add the `:py:meth:` indicator but kept the full method path so that should work fine.